install-storageos/050-persistent-volume.md

```diff
@@ -0,0 +1,106 @@
+Developers can request storage using Persistent Volume Claims. In order to
+demonstrate this we will deploy a Redis Server on Kubernetes (as Redis can
+persist its state to a persistent volume) with the data being written to a
+StorageOS persistent volume.
+
+We could use a Statefulset to deploy Redis with persistent storage. For
+didactic purposes, we are just using a PVC and Pod.
+
+Create PersistentVolumeClaim and a Redis pod which references the `prod` StorageClass:
+
+```
+kubectl create -f- <<END
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: redis-prod-pvc
+spec:
+  storageClassName: prod
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+END
+``` {{execute}}
+
+Check the PVC and PV
+
+`kubectl get pvc`{{execute}} and `kubectl get pv`{{execute}}
+
+> StorageOS dynamically provisions a PV (Persistent Volume) that binds to
+> the PVC(Persistent Volume Claim).
+
+Check the Volume in StorageOS
+
+`kubectl exec -ti cli -n kube-system -- storageos get volumes`{{execute}}
+
+Because we set the replication to 1 for the `prod` StorageClass, we can see in
+the output that the volume has `1/1` replicas. This means that the replica is healthy and ready.
```

Comment:
Many people get confused here I think we need to be even more clear. 

Because we set the replication to 1 for the `prod` StorageClass, StorageOS creates a one replica in addition to the volume primary. Hence two synchronous copies of the data. 1/1 indicates the number of replicas heathy out of the number of replicas per volume. In this case the one replica is healthy.